src/test/java/jnr/posix/FileStatTest.java

```diff
@@ -157,4 +155,12 @@ public void filestatFailed() throws Throwable {
             }
         }
     }
+
+    @Test public void statIsWritable() throws Throwable {
+      String tempDirectory = System.getenv("TEMP");
+      FileStat st = posix.stat(tempDirectory);
+
+      // FIXME: This fails on JRuby, but succeeds here. Why?
```

Comment:
Do you happen to have any idea of where that check would be? When trying to understand (the somewhat complex) JRuby part of this, I got the impression that it pretty much just forwards the call to `FileStat.isWritable()`: https://github.com/jruby/jruby/blob/master/core/src/main/java/org/jruby/RubyFileStat.java#L493